drainer/translator/mysql.go

```diff
@@ -664,3 +664,18 @@ func DecodeOldAndNewRow(b []byte, cols map[int64]*types.FieldType, loc *time.Loc
 
 	return oldRow, newRow, nil
 }
+
+// SplitWithSemicolons is used to make it compatible with some SQL
+// that cannot execute multiple statements at once
+func SplitWithSemicolons(sqls []string, args [][]interface{}) ([]string, [][]interface{}) {
+	nsqls := make([]string, 0)
+	nargs := make([][]interface{}, 0)
+	for i, sql := range sqls {
+		s := strings.SplitAfter(sql, ";")
```

Comment:
What happens when the SQL itself contains a `;`... e.g.

```sql
CREATE TABLE `foo;bar` (`col;umn` VARCHAR(20) DEFAULT 'va;lue') COMMENT = 'com;ment';
```

Reply:
Actually the program will fail. Maybe we can just check these semicolons just before some key words? Such as `USE, SELECT, CREATE, INSERT`...etc.

Reply:
Or we can check the semicolon(;) in quotation marks, like `a;b`. Regard these kinds of semicolon as a part of name.

Reply:
Or we can check the semicolon(;) in quotation marks, like \`a;b\`. Regard these kinds of semicolon as a part of name.

Reply:
it's not so simple